tests/test_translate.py

```diff
@@ -337,11 +345,11 @@ def test_parallel_savepoint(
     if testobj is None:
         pytest.xfail(f"no translate object available for savepoint {test_name}")
     # Reduce error threshold for GPU
-    if backend.endswith("cuda") and testobj.max_error < GPU_MAX_ERR:
-        testobj.max_error = GPU_MAX_ERR
+    if backend.endswith("cuda"):
+        testobj.max_error = max(testobj.max_error, GPU_MAX_ERR)
+        testobj.near_zero = max(testobj.near_zero, GPU_NEAR_ZERO)
     if threshold_overrides is not None:
         process_override(threshold_overrides, testobj, test_name, backend)
-    fv3core._config.set_grid(grid[0])
```

Comment:
ah I believe this is what is causing the tests to fail

Reply:
Thank you, I really must be more careful...